cmd/dwi2tensor.cpp

```diff
@@ -47,8 +49,12 @@ void usage ()
 
   SYNOPSIS = "Diffusion (kurtosis) tensor estimation";
 
+  AUTHOR = "Ben Jeurissen (ben.jeurissen@uantwerpen.be)";
+
+  SYNOPSIS = "Diffusion (kurtosis) tensor estimation";
+
```

Comment:
File shows both of these fields being duplicated following merge

Reply:
Thanks for spotting this